src/conf_macros.m4

```diff
@@ -895,8 +895,9 @@ AC_ARG_ENABLE([files-domain],
               [AS_HELP_STRING([--enable-files-domain],
                               [If this feature is enabled, then SSSD always enables
                                a domain with id_provider=files even if the domain
-                               is not specified in the config file
-                              [default=no]])],
+                               is not specified in the config file [default=no]
```

Comment:
Missing `.` at the end of the line

Reply:
Wouldn't `.` be weird after `[...]`?

Reply:
Oh, sorry. I thought I was reading some standard text when in reality I was reading an autoconf macro. Forget my remark.

Reply:
Well, this is actually a text to be displayed for a user (with `--help`).

Reply:
I'm just not sure if
   `is not specified in the config file [default=no].`
is better than
   `is not specified in the config file [default=no]`